copy-tiles.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# trigger the AWS Batch job to copy tiles between S3 buckets
+
+# allow overriding the values of the SOURCE_S3_PATH and DESTINATION_S3_PATH
+# env vars used by the container in the AWS Batch job
+source_s3_path="${SOURCE_S3_PATH:-s3://mbta-map-tiles-dev/osm_tiles/}"
+destination_s3_path="${DESTINATION_S3_PATH:-s3://mbta-map-tiles/osm_tiles/}"
+
+# AWS Batch job queue
+job_queue="tile-generation-prod-queue"
+# AWS Batch job definition name
+job_definition_name="tile-copy"
+# arbitrary name for the job we're submitting
+job_name="tile-copy-prod-`date "+%Y-%m-%d-%H-%M-%S"`"
+
+# get ARN of latest job definition revision
+job_definition_arn="`aws batch describe-job-definitions \
+    --job-definition-name "${job_definition_name}" \
+    | jq -r '.jobDefinitions | max_by(.revision) | .jobDefinitionArn'`"
+
+# define env vars used by container
+container_overrides="{\"environment\":[{\"name\":\"SOURCE_S3_PATH\",\"value\":\"${source_s3_path}\"},{\"name\":\"DESTINATION_S3_PATH\",\"value\":\"${destination_s3_path}\"}]}"
```

Comment:
That may be more readable if use something like `jq` to generate this json, but probably it's not a big deal.

Reply:
I fiddled with this idea for a bit, but I wasn't able to effectively reduce the complexity of what's already here, especially given the (presumed) need to escape double-quotes for passing to AWS CLI. Good idea though – I'll keep it in mind for future improvements. 😄